src/components/Chips/__tests__/chips-tests.jest.tsx

```diff
@@ -1,21 +1,55 @@
 import React from "react";
 import { render, fireEvent, screen } from "@testing-library/react";
 import Chips from "../Chips";
+import { getTestId } from "../../../tests/test-ids-utils";
```

Comment:
The reason we were not aware for the braking changes which happen because of this bug was lack of tests for this ability, so I add some...